src/condor_ce_router_defaults

```diff
@@ -50,15 +50,23 @@ JOB_ROUTER_DEFAULTS_GENERATED = \\
     set_requirements = True; \\
     /* Note default memory request of 2GB */ \\
     /* Note yet another nested condition allow pass attributes (maxMemory,xcount,jobtype,queue) via gWMS Factory described within ClassAd */ \\
-    eval_set_RequestMemory = ifThenElse(maxMemory isnt undefined, maxMemory, ifThenElse(default_maxMemory isnt undefined, default_maxMemory, 2000)); \\
+    eval_set_OriginalMemory = ifThenElse(maxMemory isnt undefined, maxMemory, ifThenElse(default_maxMemory isnt undefined, default_maxMemory, 2000)); \\
+    set_JOB_GLIDEIN_Memory = "$$(TotalMemory:0)"; \\
```

Comment:
nah - that's a match expression.  I think it dates back to ... HTCondor 6.something?  It's quite old (notice it's inside a quoted string).